client/main.js

```diff
@@ -1,23 +1,20 @@
 import React from 'react';
 import { Meteor } from 'meteor/meteor';
-import { render } from 'react-dom';
-
-import VerifyEmail from '../imports/ui/VerifyEmail';
-import {Accounts} from "meteor/accounts-base";
+import ReactDOM from 'react-dom';
+import App from '../imports/ui/App';
+import { Accounts } from "meteor/accounts-base";
+import { Session } from 'meteor/session';
 
 Meteor.startup(() => {
-	//TODO: create proper main view?
-	render(<VerifyEmail />, document.getElementById('render-target'));
+	ReactDOM.render(<App />, document.getElementById('render-target'));
 });
 
-Accounts.onEmailVerificationLink(function (token) {
-	Accounts.verifyEmail(token);
-	//TODO: create proper verify password page
+Accounts.onEmailVerificationLink((token, done) => {
+	console.log('Verifying email with token: ' + token); // eslint-disable-line
```

Comment:
Are you leaving these `console.log()`s on purpose or by accident? I don't really mind either way, just wondering